### PR TITLE
Debug Obfuscated UI

### PR DESCRIPTION
**Pull request by mureinik**
Fixes issue #431 

Changes proposed in this pull request:
- Move the "Debug Obfuscated UI" page under the developer guide, where it belongs
- Add the missing images
- Fix markdown problems

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @mureinik

This pull request needs review by: @rollandf

### CODE REVIEW

**Comment by rollandf** [comment]
:+1:

File: source/develop/developer-guide/engine/engine-debug-obfuscated-ui.html.md [hunk 1]
```diff
@@ -29,35 +29,36 @@ Why is this happening now? Recently we introduced some code that captures all ex
...
 #### Making sure we have the right symbol maps
 
 Now that we have the stack trace and the most likely culprit of the exception, we don't know what it is. When the UI is built, goes through a process of optimizing and obfuscating the original code. This is done to reduce the download size and optimize the performance of the application. The down side of this is that one can't easily tell what is going on. During this process another file is generated called a symbol map. This file maps the obfuscated code to the original source code. Due to the size of the file this is not installed by default. To install the symbol maps run the following command as root:
 
-      yum install ovirt-engine-webadmin-portal-debuginfo ovirt-engine-userportal-debuginfo
-      If you are running rhev
-      yum install rhevm-webadmin-portal-debuginfo rhevm-userportal-debuginfo
+    yum install ovirt-engine-webadmin-portal-debuginfo ovirt-engine-userportal-debuginfo
+    # If you are running rhev
+    yum install rhevm-webadmin-portal-debuginfo rhevm-userportal-debuginfo
```
Comment (mykaul) [inline source/develop/developer-guide/engine/engine-debug-obfuscated-ui.html.md]: In both cases (RHEV or oVirt), you need to restart the ovirt-engine service.

Comment (mureinik) [inline source/develop/developer-guide/engine/engine-debug-obfuscated-ui.html.md]: I just fixed the formatting in the original page, didn't delve into the content.  

But you are right, I'll add another patch to state it.

**Comment by mureinik** [comment]
PR was closed by mistake, reopened and added a patch as per @mykaul's comment.

**Comment by rollandf** [comment]
:+1:

**Pull request merged**